firstapp/executeCuj

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+PRE_CUJ=${1?ERROR: no Pre CUJ given}
+POST_CUJ=${2?ERROR: no Recorded CUJ given}
+NUM=${3:-1}
+RECORD=${4:-"n"}
+
+
+
+adb logcat -c
+sed -i -e "s/int NUM_ITERATIONS = .*;/int NUM_ITERATIONS = $NUM;/g" ./app/src/androidTest/java/com/example/firstapp/UserCujTest.java
+sed -i -e "s/static String.*preCUJ = .*;/static String[] preCUJ = $PRE_CUJ;/g" ./app/src/androidTest/java/com/example/firstapp/UserCujTest.java
+sed -i -e "s/static String.*postCUJ = .*;/static String[] postCUJ = $POST_CUJ;/g" ./app/src/androidTest/java/com/example/firstapp/UserCujTest.java
+
+if [ "$RECORD" = "r" ] 
+then 
+	sed -i -e "s/long RECORDING_BUF = .*;/long RECORDING_BUF = 1000;/g" ./app/src/androidTest/java/com/example/firstapp/UserCujTest.java
+else 
+	sed -i -e "s/long RECORDING_BUF = .*;/long RECORDING_BUF = 0;/g" ./app/src/androidTest/java/com/example/firstapp/UserCujTest.java
+fi 
+
+
+
+#./gradlew connectedAndroidTest
+./gradlew installDebugAndroidTest
+
+if [ "$RECORD" = "r" ] 
+then 
+	RECSTART=$(( $(adb shell date '+%s%N') / 1000000))
+	./../../adb shell "screenrecord /sdcard/test.mp4" &
+	PID=$!
+fi 
+
+adb shell am instrument -w -e class com.example.firstapp.UserCujTest com.example.firstapp.test/androidx.test.runner.AndroidJUnitRunner
+
+if [ "$RECORD" = "r" ] 
+then 
+	kill $PID 
+	#RECEND=$(( $(adb shell date '+%s%N') / 1000000))
+	#rm test.mp4
+	adb pull sdcard/test.mp4 
+	adb pull sdcard/test.mp4  #for some reason, the first pull sometimes doesn't always get the "moov" atom, which we need. Pulling twice seems to always do the trick...
```

Comment:
What is `moov atom`?

Reply:
"Moov atom contains the metadata and the index of the video"